src/group_theory/subgroup.lean

```diff
@@ -1662,6 +1717,143 @@ set.subset.antisymm
     by { rw sup_eq_closure, apply Inf_le _, dsimp, refl })
   ((sup_eq_closure N H).symm ▸ subset_closure)
 
-end subgroup
+@[to_additive] lemma mem_sup_iff
```

Comment:
I am not convinced yet that `mem_sup_iff` or `mem_sup_iff'` are going to be useful: in general, `(H ⊔ K : set G) = H * K` does not have to hold, and if it does, it is probably easier to use `set.mem_mul` directly. Do you have an example where you apply them?